angular-spa-ui/js/admin-app/content-manager-module/contentManager-srv.js

```diff
@@ -22,8 +22,116 @@ module.exports = function () {
         return tabs;
     };
 
+    var setVisibility = function (defaultParams, params, config) {
+
+        if (!_.isEqual(defaultParams, params)) {
+            config.visibility = true;
+        }
+
+        return config;
+    }
+
+    var setResultsCount = function (config, data) {
+
+        var from = ((data.page * data.perPage) - data.perPage + 1) || configService.getData('recordsListConfig', 'header.params.resultsCount.params.from'),
+            count = data.count || configService.getData('recordsListConfig', 'header.params.resultsCount.params.to'),
+            to = setTo() || configService.getData('recordsListConfig', 'header.params.resultsCount.params.count'),
+            params = {
+                from: from,
+                to: to,
+                count: count
+            },
+            defaultParams = configService.getData('recordsListConfig', 'header.params.resultsCount.params');
+        if (count < to || from >= to)
+            return config;
+
+        _.extend(config.params, params);
+
+        return setVisibility(defaultParams, config.params, config);
+
+        function setTo() {
+            var resultsTo = data.page * data.perPage;
+            if (resultsTo > data.count) {
+                resultsTo = data.count;
+            }
+            return resultsTo;
+        }
+    };
+
+    var setSortBy = function (config, value) {
+        var sortByValue = value || configService.getData('recordsListConfig', 'header.params.sortBy.params.value');
```

Comment:
I see so much similar long lines like configService.getData('recordsListConfig', 'header.params.sortBy.params.value'); Don't like it